src/Graphula/Class.hs

```diff
@@ -22,13 +24,26 @@ import Data.Kind (Constraint, Type)
 import Database.Persist (Entity(..), Key, PersistEntity, PersistEntityBackend)
 import Database.Persist.Sql (SqlBackend)
 import Test.QuickCheck.Random (QCGen)
+#if MIN_VERSION_persistent(2,14,0)
+import Database.Persist.Class.PersistEntity
+#endif
+
+-- | A class that provides backwards compatibility with @persistent-2.14@.
+-- If you are using that version or above, then this is a class alias for
+-- @SafeToInsert@. Otherwise, it is an identity alias.
```

Comment:
Just rewriting this in our own style:

```suggestion
-- | A class that provides backwards compatibility with @persistent-2.14@
--
-- If you are using that version or above, then this is a class alias for
-- 'SafeToInsert'. Otherwise, it is an identity alias.
--
```